res/database/Default/database_programmes_fictional.xml

```diff
@@ -759,10 +861,12 @@ Exciting entertainment and brain-teasing trivia questions.</en>
 					<title>
 						<de>Brasilien - Argentinien 0:1</de>
 						<en>Brazil 0-1 Argentina</en>
+						<pl>Brazylia - Argentyna</pl>
 					</title>
 					<description>
 						<de></de>
 						<en></en>
+						<pl>Finał</pl>
```

Comment:
Not the final. `:)`

(Although erroneously mentioned in the description, see comment. This whole "format" will eventually have to be reworked.)

Reply:
We have very few sports events. In this case, giving the results of the meeting before the meeting is broadcast is a bit strange.  
In this case, I thought we were broadcasting 2 matches from Group A and then the final. 
What to do with this?